### PR TITLE
Feature/run tasks

### PR DESCRIPTION
**Pull request by robert-bryson**
Related to https://github.com/GSA/datagov-deploy/issues/3251

`cf run-task` appear to be working correctly from my understanding. Tasks run successfully and are visible in `cf tasks`.

### CODE REVIEW

File: .profile [hunk 1]
```diff
@@ -52,4 +54,4 @@ echo "USE_LOCAL_STORAGE=true" >> $APP_DIR/.env
 
 echo "ENVIRONMENT=development" >> $APP_DIR/.env
 
-which apache2-foreground && exec "apache2-foreground" || exit 0
+#which apache2-foreground && exec "apache2-foreground" || exit 0
```
Comment (jbrown-xentity) [inline .profile:57]: Just remove this line, we don't want it lingering long term...

File: manifest.yml [hunk 2]
```diff
@@ -23,9 +23,8 @@ applications:
     # NEW_RELIC_APP_NAME and NEW_RELIC_MONITOR_MODE
     # can't be set in the PHP buildpack extension
     NEWRELIC_HOST: gov-collector.newrelic.com
-    NEWRELIC_LICENSE: ((NEWRELIC_LICENSE))
   processes:
   - type: web
-    command: $HOME/.bp/bin/start
+    #command: $HOME/.bp/bin/start
```
Comment (jbrown-xentity) [inline manifest.yml:28]: Just remove this line, we don't need it long term...

**Review by jbrown-xentity — APPROVED** [review]
Looks good, with some minor cleanup!

File: docker/Dockerfile [hunk 3]
```diff
@@ -51,8 +51,6 @@ RUN /tmp/install-composer.sh
 # Add composer-installed libs to path
 ENV PATH=/var/www/app/vendor/bin:$PATH
 
-COPY docker/apache2-foreground /usr/local/bin/
```
Comment (robert-bryson) [inline docker/Dockerfile:54]: Good catch, thanks @nickumia-reisys

**Comment by nickumia-reisys** [comment]
The tests are failing because `bootstrap.sh` no longer exists.  I don't know enough about the context of the switch to `.profile` to know if it's safe to just delete this, but just trying to help out @robert-bryson 
- https://github.com/GSA/project-open-data-dashboard/blob/feature/run-tasks/docker/startup.sh#L4

**Review by michaelstephenson0613 — COMMENTED** [review]
> @``

**Pull request merged**